flink-runtime/src/test/java/org/apache/flink/runtime/taskmanager/TaskManagerLocationTest.java

```diff
@@ -26,7 +26,7 @@
 import java.net.InetAddress;
 
 import static org.assertj.core.api.Assertions.assertThat;
-import static org.junit.jupiter.api.Assertions.fail;
+import static org.assertj.core.api.Assertions.fail;
```

Comment:
we use assertThatThrowsBy to replace `fail()`

Reply:
fixed